app/test/conversation_list/conversation_list_content_test.dart

```diff
@@ -37,10 +40,9 @@ final conversations = [
           sender: "bob@localhost",
           sent: true,
           content: UiMimiContent(
-            id: 1.messageId(),
-            timestamp: DateTime.parse("2023-01-01T00:00:00.000Z"),
-            lastSeen: [],
-            body: 'Hello Alice',
+            plainBody: 'Hello Alice',
+            topicId: Uint8List(0),
+            content: const MessageContent(content: []),
```

Comment:
What is this tested for? Should I hard code the whole content widget that's normally generated by Rust?

Reply:
@boxdot can you answer this one?

Reply:
It is a snapshot test. Renders the widgets and compares to the committed image.